pkg/ccl/changefeedccl/changefeed_stmt.go

```diff
@@ -84,7 +84,8 @@ func init() {
 
 type annotatedChangefeedStatement struct {
 	*tree.CreateChangefeed
-	originalSpecs map[tree.ChangefeedTarget]jobspb.ChangefeedTargetSpecification
+	originalSpecs         map[tree.ChangefeedTarget]jobspb.ChangefeedTargetSpecification
+	overRideStatementTime *hlc.Timestamp
```

Comment:
nit: perhaps `overrideCursor`; or even `alterChangefeedAsOf` might be better names.